.ci/test.sh

```diff
@@ -53,16 +53,24 @@ fi
 if [[ $TASK == "lint" ]]; then
     conda install -q -y -n $CONDA_ENV \
         pycodestyle \
-        pydocstyle \
-        r-lintr
+        pydocstyle
     pip install --user cpplint
     echo "Linting Python code"
     pycodestyle --ignore=E501,W503 --exclude=./compute,./.nuget . || exit -1
     pydocstyle --convention=numpy --add-ignore=D105 --match-dir="^(?!^compute|test|example).*" --match="(?!^test_|setup).*\.py" . || exit -1
+
+    # stringi and r-lintr
     echo "Linting R code"
```

Comment:
Please move this line right before `Rscript ...` call where the linting is actually happening.

Reply:
ah ok sure, can do

Reply:
moved